qutip/solver/brmesolve.py

```diff
@@ -333,33 +334,46 @@ def __init__(
         self._system = H, a_ops, c_ops
         self._num_collapse = len(c_ops)
         self._num_a_ops = len(a_ops)
-        self.rhs = self._prepare_rhs()
-        self._integrator = self._get_integrator()
-        self._state_metadata = {}
-        self.stats = self._initialize_stats()
-        self.rhs._register_feedback({}, solver=self.name)
+        self._post_init(options)
 
     def _initialize_stats(self):
         stats = super()._initialize_stats()
         stats.update({
             "solver": "Bloch Redfield Equation Evolution",
-            "init time": stats["init time"] + self._init_rhs_time,
+            "Build rhs time": 0,
```

Comment:
```suggestion
            "build_rhs_time": 0,
```
If we're going to add a new name, could we make it a valid Python identifier to be consistent with most of the other entries in `stats`.